app/models/user.rb

```diff
@@ -139,6 +139,10 @@ def can_supply_using_pgd?
     cis2_enabled? ? cis2_info.can_supply_using_pgd? : fallback_role_nurse?
   end
 
+  def can_prescribe_psd?
```

Comment:
@thomasleese On further reflection, would `can_add_psd`? be a better description because prescribing is more to do with medicines. A PSD is just permission for a HCA to carry out the vaccination.